app/views/tasks/index.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="list">
+  <h2>Your Tasks</h2>
+
+  <ol>
+    <% @tasks.each do |task| %>
+      <li>
+        <strong><%= link_to task[:name], task_path(task.id) %></strong> ||
+        description: <em><%= task[:description] %></em> ||
+        completion_date: <em><%= task[:completion_date] %></em>
+        <ul>
+          <li>
+            <% if task.status %>
```

Comment:
This `if` statement can probably be DRY'd up somewhat.

The button that we're creating in both cases is identical except for the text that it will have. Since that is the case we can probably use an `if` statement to define a variable representing the text and then write the `button_to` code once, using that variable.